backup: add experimental description for sql br (#16074)
pingcap/docs: br/backup-and-restore-tool.md

### How to use BR

Currently, the following methods are supported to use BR:

- Use the command-line tool
- Use SQL statements

> - Use BR In the Kubernetes environment

Reviewer: ```suggestion
- Use BR in the Kubernetes environment
```

#### Use the command-line tool

In TiDB versions above v3.1, you can run the BR tool using the command-line tool.

First, you need to download the binary file of the BR tool. See [download link](/download-ecosystem-tools.md#br-backup-and-restore).

For how to use the command-line tool to perform backup and restore operations, see [Use the BR command-line tool](/br/use-br-command-line-tool.md).

#### Use SQL statements

> **Warning:**
> 
> The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [issue feedback] on GitHub (/support.md).

Reviewer: ```suggestion
> **Warning:**
>
> The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [report it on GitHub](/support.md).
```

For how to use the command-line tool to perform backup and restore operations, see [Use the BR command-line tool](/br/use-br-command-line-tool.md).

#### Use SQL statements

**Warning:**

The `BACKUP` statement is still an experimental feature. It is **NOT** recommended that you use it in the production environment. This feature is subject to change or removal without notice. If you find a bug, please [issue feedback] on GitHub (/support.md).

In TiDB v4.0.2 and later versions, you can run the BR tool using SQL statements.

> For detailed operations, see the following documents:

Reviewer 2: ```suggestion

For detailed operations, see the following documents:
```

Or merge with like 189.

```
[2024-01-10T09:37:20.189Z] br/backup-and-restore-tool.md: this file has manual line breaks in the following lines:
[2024-01-10T09:37:20.189Z] 
[2024-01-10T09:37:20.189Z] MANUAL LINE BREAKS: L190
```
